dll/win32/userenv/lang/pt-PT.rc

```diff
@@ -7,7 +7,7 @@ BEGIN
     IDS_APPDATA "Dados de Programas"
     IDS_DESKTOP "Ambiente de trabalho"
     IDS_FAVORITES "Favoritos"
-    IDS_MYDOCUMENTS "Meus Documentos"
+    IDS_MYDOCUMENTS "Os Meus Documentos"
```

Comment:
Can you also adjust the one in the pt-PT translations in shell32?
https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=Meus+Document
the `IDS_PERSONAL` : "OS Meus Documentos" <--- OS in all capitals or just written as "Os" ?

Reply:
All other references to this folder in Win32 have the correct name except this one.

Reply:
The same problem persists but  in pt-BR

Reply:
I see. Thanks